packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -232,15 +234,16 @@ describe('App: Settings', () => {
 })
 
 describe('App: Settings without cloud', () => {
-  it('hides the projectId section when there is no projectId', () => {
+  it('the projectId section shows a prompt to connect when there is no projectId', () => {
     cy.scaffoldProject('simple-ct')
     cy.openProject('simple-ct')
     cy.startAppServer('component')
 
     cy.visitApp()
     cy.findByText('Settings').click()
-    cy.findByText('Project Settings').click()
-    cy.findByText('Project ID').should('not.exist')
+    cy.findByText('Dashboard Settings').click()
+    cy.findByText('Project ID').should('exist')
+    cy.contains('button', 'Log in').should('be.visible')
```

Comment:
Pretty minor, but this could be more specific since we have a "Log in" button in the header.

Reply:
good catch 👍🏻